style.css

```diff
@@ -0,0 +1,137 @@
+#nav {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 50px;
+}
+
+#nav a {
+  text-decoration: none;
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 30px;
+  color: #6070ff;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-left: 24px;
+}
+
+#nav a:hover {
+  text-decoration: underline;
+  -webkit-text-decoration-color: #23a3fa;
+  text-decoration-color: #23a3fa;
+  text-decoration-thickness: 2px;
+}
+
+#nav nav {
+  width: 45px;
+  height: 30px;
+  display: flex;
+  align-items: center;
+}
+
+section {
+  background-image:
+    url("../images/Ellipse 11.png"),
+    url("../images/Rectangle 4.png"),
+    url("../images/Ellipse\ 3.png"),
+    url("../images/Ellipse\ 8.png"),
+    url("../images/Ellipse\ 5.png"),
+    url("../images/Rectangle\ 7.png"),
+    url("../images/Ellipse 12.png"),
+    url("../images/Ellipse 4.png"),
+    url("../images/Rectangle\ 5.png"),
+    url("../images/Ellipse\ 13.png"),
+    url("../images/Ellipse\ 15.png");
+  background-position:
+    19px 79px,
+    294px 68px,
+    323px 102px,
+    214px 110px,
+    125px 106px,
+    127px 98px,
+    370px 385px,
+    290px 509px,
+    240px 507px,
+    357px 392px,
+    106px 497px;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 600px;
+}
+
+section h1 {
+  font-family: Poppins, sans-serif;
+  font-weight: bold;
+  font-size: 40px;
+  color: #172b4d;
+  line-height: 52px;
+  text-align: start;
+  margin: 0 24px;
+}
+
+section p {
```

Comment:
- [ ] [OPTIONAL] Poppins is the only font family used in this template, so it's best to declare it in your body selector to keep your css dry, as it'll be inherited from the body selctor to the other selectors.